src/main/java/rx/schedulers/NewThreadScheduler.java

```diff
@@ -16,28 +16,18 @@
 package rx.schedulers;
 
 import rx.Scheduler;
-import rx.internal.schedulers.NewThreadWorker;
-import rx.internal.util.RxThreadFactory;
 
 /**
- * Schedules work on a new thread.
+ * @deprecated This type was never publicly instantiable. Use {@link Schedulers#newThread()}.
  */
+@Deprecated
 public final class NewThreadScheduler extends Scheduler {
```

Comment:
It would be nice to just delete this type. It was `public` yes, but it was `final`, had a private constructor, `instance()` was package scoped, and `Schedulers.newThread()` was never guaranteed to return instance of it. This basically makes it completely unusable from application code.